### PR TITLE
[INLONG-9596][Sort] Support row way of sort InLong message tlog-kv format

### LINKED ISSUE #9596: [Feature][sort] Support row way of sort InLong message tlog-kv format
### Description

tlog format like this
“streamId|k1=v1|k2=v2|k3=v3”

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
Support row way of sort InLong message tlog-kv format

- Fixes #9596

### CODE REVIEW

File: inlong-sort/sort-formats/format-row/format-inlongmsg-tlogkv/src/main/java/org/apache/inlong/sort/formats/inlongmsgtlogkv/InLongMsgTlogKv.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import static org.apache.inlong.sort.formats.base.TableFormatConstants.FORMAT_DELIMITER;
+import static org.apache.inlong.sort.formats.base.TableFormatConstants.FORMAT_ENTRY_DELIMITER;
+import static org.apache.inlong.sort.formats.base.TableFormatConstants.FORMAT_KV_DELIMITER;
+
+/**
+ * Format descriptor for InLongMsgTlogKVs.
+ */
+public class InLongMsgTlogKv extends TextFormatDescriptor<InLongMsgTlogKv> {
+
+    public static final String FORMAT_TYPE_VALUE = "InLongMsg-TlogKV";
```
Comment (EMsnap) [inline inlong-sort/sort-formats/format-row/format-inlongmsg-tlogkv/src/main/java/org/apache/inlong/sort/formats/inlongmsgtlogkv/InLongMsgTlogKv.java:33]: ditto

Comment (baomingyu) [reply inlong-sort/sort-formats/format-row/format-inlongmsg-tlogkv/src/main/java/org/apache/inlong/sort/formats/inlongmsgtlogkv/InLongMsgTlogKv.java]: change to inlong-msg-tlogkv

**Review by EMsnap — APPROVED** [review]

**Review by vernedeng — APPROVED** [review]

File: inlong-sort/sort-formats/format-row/format-inlongmsg-tlogkv/src/main/java/org/apache/inlong/sort/formats/inlongmsgtlogkv/InLongMsgTlogKvUtils.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        String text;
+        if (bytes[0] == delimiter) {
+            text = new String(bytes, 1, bytes.length - 1, Charset.forName(charset));
+        } else {
+            text = new String(bytes, Charset.forName(charset));
+        }
+
+        String[] segments = splitCsv(text, delimiter, escapeChar, quoteChar);
+
+        String tid = segments[0];
```
Comment (dockerzhang) [inline inlong-sort/sort-formats/format-row/format-inlongmsg-tlogkv/src/main/java/org/apache/inlong/sort/formats/inlongmsgtlogkv/InLongMsgTlogKvUtils.java:88]: streamId？

Comment (baomingyu) [reply inlong-sort/sort-formats/format-row/format-inlongmsg-tlogkv/src/main/java/org/apache/inlong/sort/formats/inlongmsgtlogkv/InLongMsgTlogKvUtils.java:88]: next pr handle this problem all. tid is in inlong everywhere

**Review by aloyszhang — APPROVED** [review]

**Pull request merged**